Enhance permission denied exception logs (#875)
Flank/flank: docs/permissions_denied_behavior.md

```

> You can reproduce it by set PROJECT_ID to project where your current firebase account doesn't have permission.

Reviewer: ```suggestion
You can reproduce the error by setting PROJECT_ID to a project that the firebase account doesn't have permission to access.
```

```

> You can reproduce it by set PROJECT_ID to not existing project.

Reviewer: ```suggestion
You can reproduce the error by setting PROJECT_ID to a project that doesn't exist.
```
